t/24-worker-engine.t

```diff
@@ -227,13 +229,20 @@ subtest 'symlink testrepo' => sub {
     $result  = OpenQA::Worker::Engines::isotovideo::_link_repo($casedir, $pool_directory, 'opensuse');
     is $result, undef, 'create symlink successfully';
 
-    $settings->{DISTRI}   = 'mine';
-    $settings->{JOBTOKEN} = 'token99916';
-    delete $settings->{NEEDLES_DIR};
-    $settings->{CASEDIR} = 'https://github.com/foo/os-autoinst-distri-example.git#master';
-    $job                 = OpenQA::Worker::Job->new($worker, $client, {id => 12, settings => $settings});
-    $result              = OpenQA::Worker::Engines::isotovideo::engine_workit($job);
+    $settings->{DISTRI}      = 'fedora';
```

Comment:
I wonder, does it still work with the previous definition of "mine" or is there a reason why it does not work?